src/components/app-content-renderer/recommendation-tile.js

```diff
@@ -42,15 +53,43 @@ const RecommendationGroup = ({
   description,
   action,
   permissions,
+  api,
+  condition,
+  accessor,
 }) => {
+  const intl = useIntl();
+  const [{ count, data, show }, setState] = useReducer(reducer, {});
+
+  const text = (message) =>
+    typeof message === 'object'
+      ? intl.formatMessage(message, { count, data })
+      : message;
+
   const GroupIcon = groupIconMapper[icon] || NoIcon;
-  const [hasPermission, setHasPermission] = useState(false);
+
   useEffect(async () => {
     const hasPermission = await permissionProcessor(permissions);
-    setHasPermission(hasPermission);
+
+    if (hasPermission && api) {
+      try {
+        const data = await instance.get(api);
```

Comment:
Just for future-proofing. The API calls might not just be `GET` requests. Already in the estate section, we have some `POST` methods. I would use the `processRequest` method (or something similar if we can't re-use it) for data fetching.